bears/csv/CSVLintBear.py

```diff
@@ -3,9 +3,8 @@
 
 @linter(executable='csvlint',
         output_format='regex',
-        output_regex=r'\d\. (?P<message>.+)\. Row:'
-                     r' (?P<row>\d+)\. (?P<information>.+)',
-        result_message='This ``csv`` file is invalid.')
+        output_regex=r'\d\. (?P<message>.+(s|g|e|w)\.*)(?P<additional_info>('
+                     r' Row: \d+.|$| Column: \d+).*)')
```

Comment:
why spec row and column if you ignore them completely? Why ignore them completely?